integration-tests/kyuubi-kubernetes-it/src/test/scala/org/apache/kyuubi/kubernetes/test/WithKyuubiServerOnKubernetes.scala

```diff
@@ -32,6 +38,17 @@ trait WithKyuubiServerOnKubernetes extends WithKyuubiServer {
     kyuubiServerConf
   }
 
+  protected lazy val kubernetesOperation: KubernetesApplicationOperation = {
+    val operation = new KubernetesApplicationOperation()
+    operation.initialize(conf, null)
```

Comment:
not sure how test passed.. seems the client mode has no way to kill a running application since jps is null

Reply:
Confusing...I will do breakpoint debugging locally to find out why.